HATE/Shuffle.cs

```diff
@@ -453,47 +451,34 @@ public static bool ShuffleText_Shuffler(UndertaleChunk _chunk, UndertaleData dat
                 continue;
 
             string ch = "";
-            foreach (string chr in DRChoicerControlChars)
+            foreach (string chr in DRChoicerControlChars.Where(chr => convertedString.Contains(chr)))
             {
-                if (convertedString.Contains(chr))
-                {
-                    ch = chr;
-                    break;
-                }
+                ch = chr;
+                break;
             }
             if (ch != "")
             {
-                foreach (char ix in convertedString)
```

Comment:
I deliberately expanded that Any call due of unreasonable performance concerns, does Any check items until the condition is true or does it check all items in the enumerable?

Reply:
Any checks until the condition is true. If it finds an element that's true, it exists.